db/migrate/20200507034433_change_column.rb

```diff
@@ -0,0 +1,5 @@
+class ChangeColumn < ActiveRecord::Migration[5.1]
+  def change
+    change_column :pets, :adoption_status, :string, :default => "adoptable"
+  end
+end
```

Comment:
This sets default value of new pets as adoptable so that when user enters in a new pet it will be adoptable.